migrations_lockfile.txt

```diff
@@ -10,7 +10,7 @@ auth: 0008_alter_user_username_max_length
 contenttypes: 0002_remove_content_type_name
 jira_ac: 0001_initial
 nodestore: 0001_initial
-sentry: 0127_backfill_platformexternalissue_project_id
+sentry: 0128_change_dashboards
```

Comment:
Not super familiar but this says 0128, yet you've included a migration 0129?

Reply:
It should. Fixed!